src/QuartzNET-DynamoDB.Tests/Integration/DynamoClientFactory.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Amazon.DynamoDBv2;
+
+namespace Quartz.DynamoDB.Tests
+{
+    public class DynamoClientFactory
+    {
+        private readonly string _instanceName = Guid.NewGuid().ToString();
+
+        public DynamoDB.JobStore CreateTestJobStore()
+        {
+            var var = new JobStore();
+            var.InstanceName = _instanceName;
+
+            return var;
+        }
+
+        public AmazonDynamoDBClient BootStrapDynamo()
+        {
+            var client = DynamoDbClientFactory.Create();
+            DynamoConfiguration.InstanceName = _instanceName;
+            new DynamoBootstrapper().BootStrap(client);
+
+            return client;
+        }
+
+        public void CleanUpDynamo()
+        {
+            using (var client = DynamoDbClientFactory.Create())
+            {
+                DynamoConfiguration.InstanceName = _instanceName;
+
+                foreach (var table in DynamoConfiguration.AllTableNames)
+                {
+                    client.DeleteTable(table);
```

Comment:
You can't trunc a table in dynamo, so would need to use batchWriteItem. I assume this would perform worse. 

Refer: http://stackoverflow.com/questions/9154264/what-is-the-recomended-way-to-delete-a-large-number-of-items-from-dynamodb